src/py_sonar_scanner/configuration.py

```diff
@@ -37,43 +37,14 @@ def __init__(self):
         self.sonar_scanner_executable_path = ""
```

Comment:
I cannot highlight the exact line, but the `argparse` import on line 21 is no longer needed.